deploy/utils.sh

```diff
@@ -388,3 +389,57 @@ verify_secret_value_in_pod() {
   $cli_with_timeout "exec -n $APP_NAMESPACE_NAME ${pod_name} -- \
     printenv | grep $environment_variable_name | cut -d '=' -f 2- | grep $expected_value"
 }
+
+get_app_logs_container() {
+    echo "Get logs from the Secrets Provider container"
+    set_namespace "$APP_NAMESPACE_NAME"
+    helm=$(helm ls -aq)
+    $cli_without_timeout get pods
+
+    if [[ -z "$helm" ]]; then
+      pod_name=$($cli_without_timeout get pods --selector app=test-env --no-headers  | awk '{print $1}' )
+      echo "pod_name="$pod_name
+
+      if [[ $pod_name != "" ]]; then
+        $cli_without_timeout logs $pod_name -c cyberark-secrets-provider-for-k8s > "output/$SUMMON_ENV-secrets-provider-logs.txt"
+      fi
+    else
+      pod_name=$($cli_without_timeout get pods --selector app=test-helm --no-headers | awk '{print $1}' )
+      echo "pod_name="$pod_name
+
+      if [[ $pod_name != "" ]]; then
+         $cli_without_timeout logs $pod_name  > "output/$SUMMON_ENV-secrets-provider-logs-with-helm.txt"
+      fi
+    fi
+}
+
+get_conjur_logs_container() {
+    echo "Get logs from Conjur"
+    set_namespace "$CONJUR_NAMESPACE_NAME"
+    selector="role=follower"
+
+    if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+      selector="app=conjur-oss"
+    fi
+
+    $cli_without_timeout get pods
+    $cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1
+    pod_name=$($cli_without_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
```

Comment:
what if there are multiple followers?  don't we want to provide a way to fetch all logs from all followers? This can be accomplished by a for loop and removing `head -1`

Reply:
Great call.
I will add verification for READY 1/1 :

```
NAME                              READY   STATUS    RESTARTS   AGE
conjur-cli-6bfb6c7846-qdgkd       1/1     Running   0          15m
conjur-cluster-6d7b4db96f-9zm9f   1/1     Running   0          15m
conjur-cluster-6d7b4db96f-f62rj   0/1     Running   0          15m
conjur-cluster-6d7b4db96f-zhhcw   0/1     Running   0          15m
conjur-follower-5b4b4bcc-2j46f    1/1     Running   0          13m
```